ext/SparseDiffToolsZygote.jl

```diff
@@ -115,8 +116,8 @@ function SparseDiffTools.auto_vecjac(f, x, v)
     return vec(back(reshape(v, size(vv)))[1])
 end
 
-function SparseDiffTools.ZygoteVecJac(args...; autodiff = true, kwargs...)
-    VecJac(args...; autodiff = autodiff, kwargs...)
+function SparseDiffTools.ZygoteVecJac(args...; kwargs...)
```

Comment:
this should just be renamed to VecJac, and AutoZygote is the Zygote version of it (but then we want to make Enzyme etc. versions soon.